packages/neon/neon/lib/src/blocs/accounts.dart

```diff
@@ -1,16 +1,38 @@
-part of '../../neon.dart';
+part of 'blocs.dart';
 
 const _keyAccounts = 'accounts';
 
-abstract class AccountsBlocEvents {
+abstract interface class AccountsBlocEvents {
+  /// Logs in the given [account].
+  ///
+  /// When no other accounts are registered in [AccountsBlocStates.accounts] it should also [setActiveAccount].
```

Comment:
```suggestion
  /// When no other accounts are registered in [AccountsBlocStates.accounts] it will also [setActiveAccount].
```

Currently sounds like it has to be done outside the bloc.

Reply:
Yeah the difference between the comments on the interface and the implementation ...

But you're right